code1.rb

```diff
@@ -0,0 +1,20 @@
+#! user/bin/ruby
+
+
+require 'net/http'
+require 'json'
+
+link = "https://www.ruby-lang.org/"
+
+links = URI.parse(link)
+response = Net::HTTP.get_response(links)
```

Comment:
Why you have used links = URI.parse(link) and response = Net::HTTP.get_response(links)